src/object/camera.cpp

```diff
@@ -412,6 +453,33 @@ Camera::shake()
   }
 }
 
+void
+Camera::update_earthquake()
+{
+  if (!m_earthquake)
+    return;
+
+  if (m_earthquake_delay_timer.check())
+  {
+    if (m_earthquake_last_offset == 0.f)
+    {
+      m_earthquake_last_offset = m_earthquake_strength * graphicsRandom.randf(-2, 2);
+      m_translation.y += m_earthquake_last_offset;
+    }
+    else
+    {
+      m_translation.y -= m_earthquake_last_offset;
+      m_earthquake_last_offset = 0.f;
+    }
+
+    m_earthquake_delay_timer.start(m_earthquake_delay + static_cast<float>(graphicsRandom.rand(0, 1)));
```

Comment:
`graphicsRandom.randf` ?

Reply:
Adds variation to the delay.

Reply:
I mean: use randf instead of casting to float?

Reply:
Tried but that makes the delay be longer most of the time and the earthquake is slower than it should be. So I chose either slow it down by 1 second more or none.